### DISCUSSION TITLE
v3.9.0

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1746] [Announcements]
### Features

* **ui:** add <leader>uh binding to toggle foldcolumn ([#1741](https://github.com/AstroNvim/AstroNvim/issues/1741)) ([394d022](https://github.com/AstroNvim/AstroNvim/commit/394d0226279a84c8c4243bd820a937a6a9f432f2))
* **ui:** toggle foldcolumn remembers last active foldcolumnn size ([35c710b](https://github.com/AstroNvim/AstroNvim/commit/35c710b0de2eb3feb307db46e6df4e06a659fcfa))

### Bug Fixes

* **autocmds:** fix type issue ([a2caff8](https://github.com/AstroNvim/AstroNvim/commit/a2caff8a24a4e2295889971e9a17684d6a900944))
* **mappings:** fix disabling icons from mappings ([fa14481](https://github.com/AstroNvim/AstroNvim/commit/fa1448168205b8ec6838f963b59322189e50070c))
* **ui:** notify when toggling lsp semantic highlights ([#1737](https://github.com/AstroNvim/AstroNvim/issues/1737)) ([8dd7eed](https://github.com/AstroNvim/AstroNvim/commit/8dd7eedafc699628d6859b07b4cf83afdb3b20c3))
* **utils:** update utils.cmd ([9130334](https://github.com/AstroNvim/AstroNvim/commit/9130334ebc38b3f425991a2dbbc5e8f2a4c5ae5a))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.8.0...v3.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.9.0'>v3.9.0</a>.</em>

**Comment by oneslash** [comment]
Thanks for your work! ❤️
